src/main.js

```diff
@@ -166,46 +172,58 @@ async function shuttingDown(eventType, err) {
 		process.exit();
 	}
 }
-module.exports.shuttingDown = shuttingDown;
 
-const commandFiles = fs.readdirSync(path.join(__dirname, 'commands')).filter(file => file.endsWith('.js'));
-for (const file of commandFiles) {
+const locales = new Collection();
+const localeFolders = readdirSync(getAbsoluteFileURL(import.meta.url, ['..', 'locales']));
+for await (const folder of localeFolders) {
```

Comment:
I think the `await` here only matters for the second `for...of` loop inside this.
```suggestion
for (const folder of localeFolders) {
```